<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdaterForm</class>
 <widget class="QMainWindow" name="AutoUpdaterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>241</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>241</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>241</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CodeReload</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
	border: 1px solid black;
	text-align: top;
	padding: 1px;
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #fff,
		stop: 0.4999 #eee,
		stop: 0.5 #ddd,
		stop: 1 #eee );
	width: 15px;
}
QProgressBar::chunk {
	background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #78d,
	stop: 0.4999 #46a,
	stop: 0.5 #45a,
	stop: 1 #238 );
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	border: 1px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="addressTable">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>51</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="importPatterns">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Patterns File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processScan">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processSelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processRefresh">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="processList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QProgressBar" name="scanProgress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="processLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="patternsLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0 patterns loaded</string>
    </property>
   </widget>
   <widget class="QLabel" name="processIconLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>32</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <action name="actionPattern_File">
   <property name="text">
    <string>Pattern File</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSelected_Addresses_h">
   <property name="text">
    <string>Addresses (.h)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSelected_Addresses_vb">
   <property name="text">
    <string>Addresses (.vb)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSelected_Addresses_cs">
   <property name="text">
    <string>Addresses (.cs)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="autoupdaterform.qrc"/>
 </resources>
 <connections/>
</ui>
